assets/styles/stack.css

```diff
@@ -415,3 +431,10 @@ li.filetype-zip:before {
     padding: 0.5em;
   }
 }
+
+@media (min-width: 32rem) {
+  .with-sticky-footer footer {
+    margin: 0 calc(-1 * (100vw - 32rem) / 2);
+    padding: 1rem calc((100vw - 32rem) / 2 + 1rem);
```

Comment:
@ptbrowne I'm interested to know if you a less hacky way to add the box-shadow on the full width of the page but still restricts the content to a maximal width of 32rem.

Reply:
With two divs ? One with for the border, the other for the content ?

Reply:
Maybe, but I didn't succeed to place the div with the border at the right place (y position), both when the buttons are on the same line and when they are one above the other, and to have some other side effects that makes the whole page a mess.